### ISSUE TITLE
Embracing open science principles to develop/growth open access journals

### ISSUE THREAD

**Issue by fredbelliard** [open-life-science/ols-5#12]
Project Lead: @fredbelliard

Mentor: @jcolomb @Arielle-Bennett

Welcome to OLS-5! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Life Science program :tada:.

***

Week 1 (week starting 28 February 2022): Meet your mentor!
- [x] Meet mentor for 30 minutes 
- [x] Create an account on [GitHub](https://github.com)
- [x] Check if you have access to the HackMD notes set up for your meetings with your mentor
- [x] Prepare to meet your mentor(s) by completing a short homework provided in your shared notes
- [x] Complete **your own copy** of the [open leadership self-assessment](https://docs.google.com/document/d/1oQgdfj4lPnypAyb9_Ba0Zt7E8J5L6qMvuKwu0wgQsjs/edit?usp=sharing) and share it to your mentor 
  If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [x] Make sure you know when and how you'll be meeting with your mentor.

***

Before Week 2 (week starting 7 March 2022): Cohort Call (Welcome to Open Life Science!)
- [x] Attend call or catch up via YouTube
- [x] Create an issue on the [OLS-5 GitHub repository](https://github.com/fredbelliard/open-publishing-plan) for your OLS work and share the link to your mentor.
- [x] Draft a brief vision statement using your goals
  [This lesson](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/) from the Open Leadership Training Series (OLTS) might be helpful
- [x] Leave a comment on this issue with your draft vision statement & be ready to share this on the call
- [x] Check the [Syllabus](https://openlifesci.org/ols-5) for notes and connection info for all the cohort calls.

***

Before Week 3 (week starting 14 March 2022): Meet your mentor!
- [x] Meet mentor
- [ ] Look up two other projects and comment on their issues with feedback on their vision statement
- [ ] Complete this [compare and contrast assignment] about current and desired community interactions and value exchanges
- [X] Complete your Open Canvas 
- [x] Share a link to your [Open Canvas](https://docs.google.com/presentation/d/18N1ZZHCfjqSPwM9BzL61NKYFDsitm5tEjrkyJaDKjao/edit?usp=sharing) in your GitHub issue
- [ ] Start your [Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/)
- [ ] Comment on your issue with your draft Roadmap
- [ ] Suggest a cohort name at the bottom of the shared notes and vote on your favorite with a +1

***

Before Week 4: Cohort Call (Tooling and roadmapping for Open projects)
- [x] Attend call or catch up via YouTube
- [ ] Look up two other projects and comment on their issues with feedback on their open canvas.

***

Week 5 and later
- [ ] Meet mentor
- [ ] Create a GitHub repository for your project
- [ ] Add the link to your repository in your issue https://github.com/fredbelliard/open-publishing-plan
- [ ] Use your canvas to [start writing a `README.md` file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project
- [ ] Link to your README in a comment on this issue
- [ ] Add an [open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) to your repository as a file called `LICENSE.md`
- [ ] Add a [Code of Conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/) to your repository as a file called `CODE_OF_CONDUCT.md`
- [ ] Invite new contributors to into your work!

This issue is here to help you keep track of work as you start Open Life Science program. Please refer to the [OLS-5 Syllabus](https://openlifesci.org/ols-5) for more detailed weekly notes and assignments past week 4.

***

Week 6 
- [ ] Attend call or catch up via YouTube

***

Week 7
- [ ] Meet mentor

***

Week 8
- [ ] Attend call or catch up via YouTube

***

Week 9
- [ ] Meet mentor

***

Week 10
- [ ] Attend call or catch up via YouTube

***

Week 11
- [ ] Meet mentor

***

Week 12
- [ ] Attend call or catch up via YouTube

***

Week 13
- [ ] Meet mentor

***

Week 14
- [ ] Attend call or catch up via YouTube

***

Week 15
- [ ] Meet mentor

**Comment by jcolomb** [comment]
congratulation for your first GitHub issue

**Comment by Arielle-Bennett** [comment]
Well done on your first GitHub issue! Looking forward to reading your exercise responses here 🎉

**Comment by vhellon** [comment]
Excited to hear more about this project!

**Comment by sarenaz** [comment]
Hi @fredbelliard I'd love to learn more about your project. It sounds really interesting.

**Comment by vborghe** [comment]
Me too! Sounds extremely timely - looking forward to reading more about it.

**Comment by fredbelliard** [comment]
@Arielle-Bennett @jcolomb  here is the vision:
Making open science principles part of the publication processes.
I am working at the university to facilitate the open-access dissemination of the work of researchers and teachers. Working transparently increase visibility and impact. I am working openly to influence the research culture, to incentivise open collaborative working and to build a more replicable and robust science.

**Comment by fredbelliard** [comment]
> Well done on your first GitHub issue! Looking forward to reading your exercise responses here 🎉

**Comment by fredbelliard** [comment]
The link to the Open canvas 
https://docs.google.com/presentation/d/18N1ZZHCfjqSPwM9BzL61NKYFDsitm5tEjrkyJaDKjao/edit?usp=sharing

Feedback very welcome

**Comment by jcolomb** [comment]
seeing you are mastering GitHub now ! congrats.

The vision is a bit too vague to my taste. it sounds like you would work in an open access "büro"  to convince researchers to publish open access. (And that is not what you do, right?)

**Comment by Arielle-Bennett** [comment]
I agree with @jcolomb - one way to think about it is what your project will look like if it is 100% successful - what happens then? How will the world be changed?

**Comment by fredbelliard** [comment]
> seeing you are mastering GitHub now ! congrats.
> 
> The vision is a bit too vague to my taste. it sounds like you would work in an open access "büro" to convince researchers to publish open access. (And that is not what you do, right?)

you are correct- I'll tweak the vision

**Comment by fredbelliard** [comment]
I have updated the Open Canvas (UVP)- 
https://docs.google.com/presentation/d/18N1ZZHCfjqSPwM9BzL61NKYFDsitm5tEjrkyJaDKjao/edit?usp=sharing

**Comment by fredbelliard** [comment]
@jcolomb @Arielle-Bennett The updated vision

I want to build a transparent publishing workflow where journals users share openly their data, software and reviewers’ comment to maximize research impact through citations and increase the visibility of researchers through recognition. I want to influence the way research is conducted and communicated.
